### PR TITLE
Feature/pm 572 add new role admin

### PR DESCRIPTION
**Pull request by denisgranha**
* Added isPlayToken constant
* new role admin
* updated addresses
* migration script accepts parameter admins
* added issuance script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Then run:
 npm install
 npm run compile
 npm run migrate -- --network rinkeby
+# if you want to deploy the markets with another account (recommended), you will need to add those accounts as admins
+npm run migrate -- --network rinkeby --admins=address1,...,address2 (comma separated)
```
Comment (cag) [inline README.md]: I wonder if we should throw the `--reset` flag in there as well: I think I had to do so when deploying to Rinkeby.

Also, this means that GnosisDB should be set up before setting up these contracts in Apollo, right?

File: README.md [hunk 2]
```diff
@@ -51,3 +53,8 @@ You have now the addresses for AddressRegistry and Olympia Token, should be some
 OlympiaToken: 0x2924e2338356c912634a513150e6ff5be890f7a0
 AddressRegistry: 0x12f73864dc1f603b2e62a36b210c294fd286f9fc
 ```
+
+# Issue tokens
+```sh
+truffle exec scripts/issue_tokens.js --network=rinkeby --amount 1e18 --to <comma separated addresses>
```
Comment (cag) [inline README.md]: I should really use `truffle exec` more...

File: README.md (hunk 1 above)
Comment (denisgranha) [reply README.md]: Yeah, the first step should be create the contracts, then set up GnosisDB

File: README.md (hunk 2 above)
Comment (denisgranha) [reply README.md]: Yeah, I just discovered today, wanted to add this command here, I think it's useful, we can add also another one for allowTransfers and addAdmin

**Pull request merged**